modules/ROOT/pages/message-type-target-at-host.adoc

```diff
@@ -1,8 +1,18 @@
 == Target at Host Message Type
 
+:message-type: Target at Host
+
 The Target at Host message type defines the way in which Anypoint Partner Manager transforms inbound transactions sent from your organization to your backend applications.
 
-Use the following settings to define JSON or XML settings for the Target at Host message type:
+You can define the CSV, JSON, or XML format for the Target at Host message type.
+
+== CSV
+
+include::partial$csv-inbound-partial.adoc[]
```

Comment:
Since this is "Target" - we should be showing the settings for csv-outbound-parial.adoc. Parameters like Separator, Quotes etc are not shown in the configuration.